students/JasneetChandok/Lesson04/mailroom2.py

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env python3.6
+
+'''Lesson 3 - Mailroom Part 1'''
+
+
+# Importing Systems Module
+import sys
+
+# Starter list of existing donors
+donors = {'jasneet paramjit': ("jasneet paramjit", [50, 200, 500]), 

```

Comment:
You actually don't need to have the tuple as the value anymore, it just adds more clutter. You can just have the array as the value:

{'jasneet paramjit': [50, 200, 500], 'simran kaur': [100, 200, 500]}

This will make it slightly easier when grabbing/updating the values